.prettierignore

```diff
@@ -5,5 +5,9 @@ storybook-static
 *readme.md
 
 # Ignore Stencil Build files
-src/components.d.ts
+packages/web-components/src/components.d.ts
 src/globals.d.ts
```

Comment:
I don't actually think that this file exists?

Reply:
oo looks like we p should have updated this during the monorepo move